Source/Core/Transforms.js

```diff
@@ -403,13 +500,21 @@ define([
      * var roll = 0.0;
      * var transform = Cesium.Transforms.aircraftHeadingPitchRollToFixedFrame(center, heading, pitch, roll);
      *
-     * @private
```

Comment:
Can you update CHANGES.md including, these functions, which I had to remove from CHANGES.md for 1.23?